### ISSUE TITLE
Feature request: DiskIO without iotime calculation (was: ... zeroes for mdX devices)

### ISSUE THREAD

**Issue by osctobe** [jaor/xmobar#390]
DiskIO uses secors/io_time for showing disk i/o and so displays something closer to peak traffic in last period and not an average for the period. This fails to provide useful readings (gives only zeroes) for e.g. md that does not account io time:
```
# grep md1 /proc/diskstats ; sleep 1; grep md1 /proc/diskstats
   9       1 md1 81344 0 **4409613 0** 1117528 0 **11282416 0** 0 0 0 0 0 0 0
   9       1 md1 81344 0 **4409613 0** 1117531 0 **11282440 0** 0 0 0 0 0 0 0
```

**Comment by jaor** [comment]
well, to be pedantic: when available, actually what the monitor displays
is (bytes read/write)/(time spent reading/writing) during the period
between two monitor updates, which is the average speed for the time
spent in io.  but of course you are right that that computation makes no
sense for devices that don't provide the time spent in io!  we could use
here the time between updates to compute a speed, but that's most often
going to be much lower than the real average speed, since often the
majority of the time between updates there will be no io activity.

maybe there's other places where we could collect the timing info in
this case?

**Comment by osctobe** [comment]
In my setup I switched to mdX_crypt that I have over the md's, but for md itself I don't see any other source except maybe summing/averaging it's component disks.

I would be nice to have a mode that doesn't include the iotime in calculations and so show 'bytes r/wd in last period' or, divided by the period: 'bytes per sec r/wd in last period'.

**Comment by jaor** [comment]
> I would be nice to have a mode that doesn't include the iotime in
> calculations and so show 'bytes r/wd in last period' or, divided by
> the period: 'bytes per sec r/wd in last period'.

we can just add new template variables for read/written bytes: i've done
that in master, with names totalb, readb, writeb... does that work for you?

**Comment by jaor** [comment]
Fixed via 7aeb01c. Please reopen if that doesn't work for you, thanks!
